flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/InteractionsCountingTaskManagerGateway.java

```diff
@@ -60,4 +72,12 @@ int getSubmitTaskCount() {
 	int getInteractionsCount() {
 		return cancelTaskCount.get() + submitTaskCount.get();
 	}
+
+	void waitAllTasksSubmitted(){
+		try{
+			submitLatch.await();
+		}catch (InterruptedException e){
+			Thread.currentThread().interrupt();
+		}
+	}
```

Comment:
whitespace between braces and other symbol.

Reply:
I'm sorry for that, i'll correct it!